Tests/UnitTests/Misc/SystemInfoTests.swift

```diff
@@ -66,20 +67,6 @@ class SystemInfoTests: XCTestCase {
         expect(try SystemInfo.withReceiptResult(.nilURL).isSandbox) == false
     }
 
-    func testUseStoreKit2IfAvailable() throws {
```

Comment:
This test was simply checking that the property was set in the constructor.
One could argue that can indeed break, but at some point we have to have some trust that the basic things are being done 😅 We can't have tests for every single constructor and property.